setup.py

```diff
@@ -64,7 +64,7 @@
 
 base_dir = os.path.dirname(os.path.abspath(__file__))
 
-_date = '20240322'
+_date = '20240403'
```

Comment:
if you use a commit from 04/03, we should use the libtpu date at 04/04 because the libtpu was build midnight.

Reply:
Sounds good. Updated.

Resnet numbers still look good
```
 Training Device=xla:0/0 Epoch=1 Step=1340 Loss=0.00141 Rate=1769.13 GlobalRate=923.65 Time=20:39:43
| Training Device=xla:0/1 Epoch=1 Step=1360 Loss=0.00140 Rate=1769.17 GlobalRate=938.59 Time=20:39:45
| Training Device=xla:0/3 Epoch=1 Step=1360 Loss=0.00140 Rate=1769.14 GlobalRate=934.33 Time=20:39:45
| Training Device=xla:0/2 Epoch=1 Step=1360 Loss=0.00140 Rate=1769.13 GlobalRate=936.62 Time=20:39:45
```